python/GafferSceneUI/CameraUI.py

```diff
@@ -178,40 +367,33 @@
 
 			"description",
 			"""
-			When using "Aperture and Focal Length" control mode, this defines the X and Y dimensions
-			of the aperture.
-
-			Note that in CG applications, "Film Back" and aperture are used interchangeably - lenses
-			are treated as perfectly symmetric boxes, with the aperture at the front the same size
-			as the film back would be in a physical camera.
-
-			Once you have aperture set to match a physical camera, you can use focal length to
-			control the field of view, the same as you would with a physical camera.
-
-			The units of aperture do not matter, as long as you use the same units for focal length.
-			( By convention, millimeters ).
-
-			A set of presets are provided matching some of the cameras frequently used as references,
-			and some common cameras which are currently used, or you can use "Custom" to choose your
-			own.
-
-			Note that the final field of view of a render from this camera will depend both on the
-			camera settings, but also the resolution and film fit mode in the render settings.
+			The width and height of the aperture. Use this in
+			conjunction with the focal length to set the camera's
+			projection. It can be a measure in any unit of length, as
```

Comment:
I chose _projection_ for a few reasons:

I was thinking that users familiar with photography would prefer to think in terms of focal length, not field of view or angle of view. Maybe I'm coloured by personal experience – when I dabbled in amateur photography, I don't remember FoV being a frequent term.

There's also the potential to muddy things by saying that the result of aperture is a FoV. As you mentioned above, one of the salient points is that Gaffer always stores projection as `aperture` and `focalLength` parameters. So, the result, as it were, of `aperture` should be the specific projection of the camera.

Speaking of it more generally as projection also leaves the door open to show the correlation to `aperture` in orthographic cameras, where field of view isn't relevant.

Reply:
I guess I was thinking more along the lines that there are 3 related quantities we surface in the UI that affect the resultant frustum for a perspective projection:  Aperture, Focal Lenth and FoV. You can only change two of those at any time as one is always a product of the other two.

Aside from any more abstract meaning of those terms in the broader topic of photography - it seemed that in the original text (for this and other tool tips in this section), the use of FoV in this was about highlighting that interaction. By switching to projection does it loosen that link as it is it is anchoring against a fourth term, instead of one of the other, previously referenced properties? 

Perhaps more importantly though, is that we already have a first-class 'projection' plug on the Camera node? Stating:

>  use this in conjunction with the focal length to set the camera's projection.

Could be confusing for the reader as the 'Projection' plug (persp/ortho) is entirely unaffected by any changes here?

Reply:
I'm with Tom here I'm afraid : the original text was more comprehensive and more precise. I'm concerned that in an attempt to make things "friendlier" we're actually just making them vaguer and less complete. I think articles and tutorials are the place to be less formal and more hand-holdy, but the metadata needs to be about cold hard facts, precisely communicated.